.gitignore

```diff
@@ -1,3 +1,10 @@
+.vscode
```

Comment:
Can you remove this line from the PR? It's a system-specific ignored file and should be in people's global gitconfig (this also why there is no `.DS_Store` in this file even though I'm running on macOS at the moment).